app/code/Magento/Sales/Controller/Adminhtml/Order/Invoice/AddComment.php

```diff
@@ -39,29 +46,38 @@ class AddComment extends \Magento\Sales\Controller\Adminhtml\Invoice\AbstractInv
      */
     protected $resultRawFactory;
 
+    /**
+     * @var InvoiceRepositoryInterface
+     */
+    private $invoiceRepository;
```

Comment:
Sorry, man py fault.
I didn't notice that the parent class has a weaker access level of the same property.
Could I ask you to change it back to **protected**?
Thanks.

Reply:
Have fixed it, thank you.